ext/LoggersExt/mlflow.jl

```diff
@@ -0,0 +1,64 @@
+using MLFlowClient: MLFlow, logparam, logmetric,
+                    createrun, MLFlowRun, updaterun,
+                    logartifact, getorcreateexperiment
+
+struct MLFlowLogger
+    client::MLFlow
+    experiment_name::String
+    artifact_location::Union{String, Missing}
+end
+mlflow_logger(base_uri::String, experiment_name::String,
+    artifact_location::Union{String, Missing}) =
+    MLFlowLogger(MLFlow(base_uri), experiment_name, artifact_location)
+
+function _logmodelparams(client::MLFlow, run::MLFlowRun, model::Model)
+    model_params = params(model) |> flat_params |> collect
+    for (name, value) in model_params
+        logparam(client, run, name, value)
+    end
+end
+
+function _logmachinemeasures(client::MLFlow, run::MLFlowRun, measures::Vector{T},
+                             measurements::Vector{Float64}) where T<:Measure
```

Comment:
In this you appear to be unecessarily restricting the types, and this means less generic code. I mean, does MLFlowClient really care if a measurement is a `Float64`? If we are using a GPU, we might have `Float32` here, for example. And a measure like `true_postitive` returns an `Int`. Also, in MLJ, some measurement are not even numerical: we could have a float vector, class dictionary,  or a confusion matrix. 

Try to make this a generic as MLFlowClient will allow. If I understand rightly, MLFlowClient just calls `string` on everything. If that  is the case, I'd just leave this untyped (as in my suggestion). 

With this in mind, here are some non-standard classification measures to add in your tests: `true_positive` (integer-valued, data must be `OrderedFactor{2}`), `ConfusionMatrix()` (returns a special type wrapping a matrix), `MulticlassFScore(average=MLJBase.NoAvg())` (dictionary-valued). 

```suggestion
function _logmachinemeasures(client::MLFlow, run::MLFlowRun, measures, measurements)
```